site/content/extend/plugins/community-plugin-marketplace.md

```diff
@@ -0,0 +1,79 @@
+
+**Security Requirements (Checked by a member of the security team)**
+
+1. The plugin does not expose a vulnerability.
+2. The plugin does not include favor the author of the plugin or a third party excessively by e.g. including a bitcoin miner that mines on behalf of the author.
+3. The plugins author must notify Mattermost about any vulnerabilities in the future.
+
+**Functional Requirements (Checked by a and be avaiable  QA tester)**
+
+1. The plugin works as expected with the latest version of Mattermost.
+2. The plugin works as expected with the latest ESR version of Mattermost. This must not be checked if `min_server_version` is higher than the latest ESR version.
+
+Given that plugins are currently in Beta and rapid development is still happening, breaking changes especially for the webapp can happen. It is expected from plugin authors to keep up with these changes and fix bugs that may occur. Breaking changes will be mentioned in the Important Upgrades Notes section (CAN WE LINK THERE?) of the [Mattermost Changelog](https://docs.mattermost.com/administration/changelog.html).
+
+### Requirements for updating a community plugin to the Marketplace
+On every update of a community plugin must still fulfill the following checklist to be added to the Marketplace. This is checked by the four reviewers in the same way as when the plugin was added. The code review and security review should be performed against the `diff` of the last version in the Marketplace and the new version.
+
+The release also has to follow [semver](https://semver.org/). This specifically means for plugins:
+
+* If the plugin exposes an API for inter plugin communication, breaking changes to the API require a major version bump.
+* If an update requires manual migration actions from the administrator, a major version bump is required.
+
+This is checked in dev review.
+
+The new release must not change the plugin id defined in the manifest as this would require a reconfiguration of the plugin by a system admin.
+
+### Process for adding community plugin to the Marketplace
+There will be an owner for community plugins, that will guide you through the review process. Currently this is [hanzei](https://github.com/hanzei).
+
+1. You open an issue on [The Plugin Marketplace Repository](https://github.com/mattermost/mattermost-marketplace) using a pre defined template for new plugins. The template contains the checklist above, so you can check the items. Please also points out which commit should be used for the review. You may cut an release candidate (RC) and for it the reviews.
+2. The reviews are requested by the owner. This is done by cloning [this](TODO) JIRA ticket for every reviewer, assigning it and mentioning the ticket on GitHub. The reviewers point out found issues on the bug tracker of the community plugin. Once all issues are resolved, they comment on the issue stating the positive review and mark the JIRA ticket as resolved.
```

Comment:
Todo: Add link to JIRA ticket once https://community-release.mattermost.com/core/pl/jhy8ih9igty4fyj97nxfoinaqw is resolved.